docs/deployment/nginx.rst

```diff
@@ -220,6 +220,10 @@ Running the pserve processes::
     pserve --daemon --pid-file=pserve_5000.pid production.ini http_port=5000
     pserve --daemon --pid-file=pserve_5001.pid production.ini http_port=5001
 
+.. note::
+
+   Starting the pserve process with ``--daemon`` is no longer supported in Pyramid 1.9 and newer.
+
```

Comment:
@prcutler welcome to becoming a contributor!

I'm not sure if you know, the Cookbook is not official documentation but is community based. Where applicable, the Cookbook should refer to the official Pyramid (or whatever package) documentation. Sorry if this wasn't clear up front.

In this case, please remove the ``--daemon`` argument from the ``pserve`` launch commands, and update the new note to something like the following:

> Daemonization of pserve was [deprecated in Pyramid 1.6](https://docs.pylonsproject.org/projects/pyramid/en/latest/whatsnew-1.6.html#deprecations), then [removed in Pyramid 1.8](https://docs.pylonsproject.org/projects/pyramid/en/latest/whatsnew-1.8.html#backwards-incompatibilities).